src/code/views/share-dialog-view.js

```diff
@@ -20,7 +20,7 @@ import {ShareLoadingView} from './share-loading-view'
 // of the react function, "tr".
 import translate  from '../utils/translate'
 import socialIcons  from 'svg-social-icons/lib/icons.json'
-import { getLegacyUrl } from '../utils/s3-share-provider-token-service-helper'
+// import { getLegacyUrl } from '../utils/s3-share-provider-token-service-helper'
```

Comment:
Shouldn't you just delete this import line?

Reply:
I thought perhaps its presence might be meaningful to someone ("Hmmm, I wonder why that's not used?") but that's probably not a good reason to keep it around. I'll remove it.